apps/server/src/modules/deletion/controller/dto/deletion-request.response.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { deletionRequestFactory } from '../../domain/testing/factory/deletion-request.factory';
```

Comment:
Please try to shorten this import as pointed out also in one of the other test files (e.g., to have just `../../domain/testing`).

Reply:
done